### PR TITLE
standar io messages will have specific value produced events

### PR DESCRIPTION
**Pull request by colombod**
stdout -> StandardOutputValueProduced
sterror -> StandardErrorValueProduced

those events will be mapped to Stream jupyter messages

### CODE REVIEW

File: Microsoft.DotNet.Interactive.Jupyter/ExecuteRequestHandler.cs [hunk 1]
```diff
@@ -106,30 +95,64 @@ public async Task Handle(JupyterRequestContext context)
 
             CreateDefaultFormattedValueIfEmpty(formattedValues, value);
 
-            PubSubMessage executeResultData;
+            Queue<PubSubMessage> dataMessage = new Queue<PubSubMessage>();
```
Comment (jonsequitur) [inline Microsoft.DotNet.Interactive.Jupyter/ExecuteRequestHandler.cs]: Feel free to use `var`.

Comment (colombod) [reply Microsoft.DotNet.Interactive.Jupyter/ExecuteRequestHandler.cs]: will do!

File: WorkspaceServer/Kernel/CSharpKernel.cs [hunk 2]
```diff
@@ -246,6 +248,30 @@ public Task<bool> IsCompleteSubmissionAsync(string code)
                     output,
                     command,
                     formattedValues));
```
Comment (jonsequitur) [inline WorkspaceServer/Kernel/CSharpKernel.cs]: We should stop publishing this event here. The Jupyter layer should decide how to handle StandardOutputValueProduced.

Comment (colombod) [reply WorkspaceServer/Kernel/CSharpKernel.cs]: makes sense

**Pull request merged**